contracts/abstracts/Controlable.sol

```diff
@@ -44,21 +54,21 @@ abstract contract Controlable is AccessControlUpgradeable {
     return true;
   }
 
-  function _changeTransactionFee(uint32 transactionFee_) internal returns (bool success) {
+  function _changeTransactionFee(uint32 transactionFee_) internal onlyAdmin returns (bool success) {
```

Comment:
since you already added the modifier on the external function, we don't need to add it to the internal function